### ISSUE TITLE
Will HTTP range requests lead to many small files on disk?

### ISSUE THREAD

**Issue by sequix** [dragonflyoss/Dragonfly#1334]
I am trying to integrate [stargz-snapshotter](github.com/containerd/stagz-snapshotter) with dragonfly. It works, but I wonder **will HTTP range requests lead to many small files on disk?**

**Comment by lowzj** [comment]
No, dfdaemon will delete the temporary file on disk when transferring its content.
